Feature/improve industry data (#2513)
mercadopago/devsite-docs: guides/checkout-api-v2/industry-data/hospitality.pt.md

# Hospitalidade

Estes são os dados específicos para a indústria de **hospitalidade** que podem ser adicionados à sua integração para melhorar a aprovação dos pagamentos.

## Campos para enviar
Adicione qualquer informação extra que considerar necessária.

### Sobre itens

| Array `items` | Tipo | Descrição |
| --- | --- | --- |
| `id` | String | Código de identificação do item. |
| `title` | String | Nome do item. |
| `category_id` | String | Categoria do item. |

> | `quantity` | Integer | Quantidade de unidades para o item. |

Reviewer: ```suggestion
| `quantity` | Integer | Quantidade de unidades do item. |
```

| `unit_price` | Float | Preço unitário atribuído ao item. Pode ser um valor inteiro ou em casas decimais. Observe que as integrações para o Chile não suportam casas decimais. |

> | `event_date` | Date |Data do evento. |

Reviewer: ```suggestion
| `event_date` | Date | Data do evento. |
```

> | `category_descriptor` | Object | Descrição de la categoría. |

Reviewer: ```suggestion
| `category_descriptor` | Object | Descrição da categoria. |
```

| `last_name` | String | Sobrenome do comprador. |
| `identification` | Object | Dados de identificação do comprador. |
| `type` | String | Tipo de identificação. Está dentro do objeto `identification`. |
| `number` | String | Número de identificação. Está dentro do objeto `identification`. |
| `phone` | Object | Telefone do comprador. |
| `area_code` | Integer | Código de área do comprador. Está dentro do objeto `phone`. |
| `number` | Integer | Número de telefone do comprador. Está dentro do objeto `phone`. |
| `address` | Object | Dados do endereço do comprador. |
| `zip_code` | String | Código postal do comprador. Está dentro do objeto `address`. |
| `street_name` | String | Nome da rua do endereço do comprador. Está dentro do objeto `address`. |
| `street_number` | Integer | Número da rua do endereço do comprador. Está dentro do objeto `address`. |
| `authentication_type` | Enum | Tipo de autenticação. Pode ser "Gmail", "Facebook", "Web Nativa" ou "Outro". |
| `registration_date` | Date | Data de registro do comprador no site. |
| `is_prime_user` | Boolean | Indica se o usuário é premium. Coloque `true` se for ou `false` se não for. |

> | `is_first_purchase_online` | Boolean | Indica se é a primeira compra do cliente. Coloque `true` se for ou `false` se não for. |

Reviewer 2: ```suggestion
| `is_first_purchase_online` | Boolean | Indica se é a primeira compra do cliente. Caso seja, use `true` e, caso não seja, use `false`. |
```
